L1Trigger/L1TTrackMatch/test/L1TrackObjectNtupleMaker.cc

```diff
@@ -994,6 +1241,119 @@ void L1TrackObjectNtupleMaker::endJob() {
   delete m_trkfastjetExt_truetp_sumpt;
 }
 
+template <typename T>
```

Comment:
Do these functions need to be in this ntuple maker or can you do this in your secondary analysis as typically there aren't specific signatures being targeted by this ntuple maker

Reply:
I can remove the isHtoMu and isHtoB as they're not needed anymore. The isHard function is just checking pythia flags which is agnostic to the signature, so I'd like to keep that one function